recipe/meta.yaml

```diff
@@ -63,24 +63,24 @@ outputs:
         - cudatoolkit      # [(cuda_compiler_version or "").startswith("11")]
         - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
         - nccl             # [cuda_compiler_version != "None"]
-        - librmm ={{ rapids_version }}  # [linux]
+        - librmm >={{ rapids_version }}  # [linux and cuda_compiler_version != "None"]
       run:
         - llvm-openmp  # [osx]
-        - librmm ={{ rapids_version }}  # [linux]
+        - librmm >={{ rapids_version }}  # [linux and cuda_compiler_version != "None"]
         - __cuda  # [cuda_compiler_version != "None"]
 
   - name: _py-xgboost-mutex
     version: 2.0
     build:
-      string: cpu_0  # [cuda_compiler_version == "None"]
-      string: gpu_0  # [cuda_compiler_version != "None"]
+      string: rapidsai_cpu_0  # [cuda_compiler_version == "None"]
+      string: rapidsai_gpu_0  # [cuda_compiler_version != "None"]
 
   - name: py-xgboost
     script: install-py-xgboost.sh  # [not win]
     script: install-win-wrapper.bat  # [win]
     build:
-      string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
-      string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
+      string: rapidsai_cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+      string: rapidsai_cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
```

Comment:
Do the changes to using RMM in `libxgboost` impact the Python packages that link to it?

Reply:
Yes, since the Python package loads `libxgboost.so` at the import time.

Reply:
Thanks Philip! 🙏

Does the Python package use functions that change ABI?

Reply:
I don't think so. The Python package uses `ctypes` to call the C API functions from `libxgboost`.

Reply:
Ah ok. So in that case, think we may be able to drop the `rapidsai_` part of the build string from the Python packages. Unless they are using RMM directly

Reply:
On second thought maybe we can leave this as-is. It might be confusing to have some packages have the build string and others not